### ISSUE TITLE
[Feature] add light theme / high contrast theme

### ISSUE THREAD

**Issue by fleischsalatinspace** [goryn-clade/pathfinder#87]
Add a light / high contrast theme which is selectable in user preferences

**Comment by fleischsalatinspace** [comment]
CC @TyrHeimdalEVE i talked with sam and you maybe have some work done here?

**Comment by TyrHeimdalEVE** [comment]
@fleischsalatinspace Yes, you can test it on: https://pathfinder.eveapp.space/ (example: https://i.imgur.com/Q2XzWoo.png)

However I do want to look into revamping the theme a bit. For example the whole grey on grey and more grey is very ... monotone.

For example, as I do - change the text to white works wonders for starters. I've also done some work on the nodes themselves to make them more compact and given them other color schemes, as it will be easier to get bigger chains. And also outlined the "tentacles" a bit by changing the color scheme.

That being said, I'm no UX designer by any means so input from people would be appreciated.
